agent/src/main/java/com/microsoft/hydralab/agent/config/TestRunnerConfig.java

```diff
@@ -34,13 +35,13 @@ public class TestRunnerConfig {
     );
 
     @Bean
-    public EspressoRunner espressoRunner(DeviceManager deviceManager, TestTaskEngineService testTaskEngineService) {
-        return new EspressoRunner(deviceManager, testTaskEngineService);
+    public EspressoRunner espressoRunner(DeviceManager deviceManager, TestTaskEngineService testTaskEngineService, ADBOperateUtil adbOperateUtil) {
```

Comment:
Please revert this as I have another PR fix this: [PR](https://github.com/microsoft/HydraLab/pull/186)